File: js/programmers/level.0/prgm_120907_2.js
function solution(quiz) {
  let result = [];
  quiz.forEach((val) => {
    const [x, operator, y, equal, z] = val.split(" ");
    let sum = 0;
    if (operator === "+") {
      sum = Number(x) + Number(y);
    } else {
      sum = Number(x) - Number(y);
    }
    sum === Number(z) ? result.push("O") : result.push("X");
  });
  return result;
}

File: python/programmers/level.0/prgm_120907_2.py
def solution(quiz):
    result = []
    for q in quiz:
        left, right = q.split(" = ")
        x, operator, y = left.split()
        sum = 0

        if operator == "+":
            sum = int(x) + int(y)
        else:
            sum = int(x) - int(y)
        
        result.append("O") if sum == int(right) else result.append("X")

    return result